python/doc/examples/reliability_sensitivity/reliability/plot_nais.py

```diff
@@ -0,0 +1,248 @@
+"""
+Non parametric Adaptive Importance Sampling (NAIS)
+==================================================
+"""
+# %%
+#
+# The objective is to evaluate a probability from the Non parametric Adaptive Importance Sampling (NAIS) technique.
+#
+# We consider the four-branch function :math:`g : \mathbb{R}^2 \rightarrow \mathbb{R}` defined by:
+#
+# .. math::
+#   \begin{align*}
+#   g(X) = \min \begin{pmatrix}5+0.1(x_1-x_2)^2-\frac{(x_1+x_2)}{\sqrt{2}}\\
+#   5+0.1(x_1-x_2)^2+\frac{(x_1+x_2)}{\sqrt{2}}\\
+#   (x_1-x_2)+ \frac{9}{\sqrt{2}}\\
+#   (x_2-x_1)+ \frac{9}{\sqrt{2}}
+#   \end{pmatrix}
+#   \end{align*}
+#
+# and the input random vector :math:`X = (X_1, X_2)` which follows a Normal distribution with independent components, and identical marginals with zero mean and unit variance:
+#
+# .. math::
+#   \begin{align*}
+#   X \sim  \mathcal{N}(\mu = [0, 0], \sigma = [1,1], cov = I_2)
+#   \end{align*}
+#
+# We want to evaluate the probability:
+#
+# .. math::
+#   \begin{align*}
+#   p = \mathbb{P} \{ g(X) \leq 0 \}
+#   \end{align*}
+#
+
+# %%
+# First, import the python modules:
+
+# %%
+import openturns as ot
+from openturns.viewer import View
+import math
+
+# %%
+# Create the probabilistic model :math:`Y = g(X)`
+# -----------------------------------------------
+
+# %%
+# Create the input random vector :math:`X`:
+
+# %%
+X = ot.RandomVector(ot.Normal([0.]*2, [1]*2, ot.IdentityMatrix(2)))
```

Comment:
`X = ot.RandomVector(ot.Normal([0.0]*2, [1.0]*2, ot.IdentityMatrix(2)))`
I prefer to make floats visible for the following reasons:
  * when you read code late at night, `1.` is much less explicit than `1.0`
  * it allows to avoid confusion between float arguments and integer arguments

Reply:
@sofianehaddad Indeed...

Reply:
Thanks. It has been done.